src/features/defi/providers/fox-farming/components/FoxFarmingManager/Overview/FoxFarmingDetails.tsx

```diff
@@ -0,0 +1,91 @@
+import { Center, Flex, ModalBody, ModalFooter, Skeleton, Stack, Tag } from '@chakra-ui/react'
+import { DefiParams, DefiQueryParams } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import { foxAssetId } from 'features/defi/providers/fox-eth-lp/constants'
+import { matchPath } from 'react-router'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+import { CircularProgress } from 'components/CircularProgress/CircularProgress'
+import { Text } from 'components/Text'
+import { useBrowserRouter } from 'hooks/useBrowserRouter/useBrowserRouter'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { useFoxFarmingBalances } from 'pages/Defi/hooks/useFoxFarmingBalances'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { FoxFarmingEmpty } from './FoxFarmingEmpty'
+import { WithdrawCard } from './WithdrawCard'
+
+export const FoxFarmingDetails = () => {
+  const { opportunities, loading } = useFoxFarmingBalances()
+  const {
+    query,
+    history: browserHistory,
+    location: browserLocation,
+  } = useBrowserRouter<DefiQueryParams, DefiParams>()
+  const match = matchPath<DefiParams>(browserLocation.pathname, {
+    path: '/defi/:earnType/:provider/:action',
```

Comment:
Again not needed to change this here, but we should probably be able to have more accurate path matching in all DeFi providers since we know which provider we're checking against

Reply:
https://github.com/shapeshift/web/issues/2434